system/jlib/jthread.cpp

```diff
@@ -874,6 +874,12 @@ class CPooledThreadWrapper: public Thread
     {
         parent.numrunning++;
     }
+    virtual void start() override
```

Comment:
Arguably a better fix is to move the capture of logctx from Thread::start into Thread::Thread constructor